NuimoSDK/NuimoGestureEvent.cs

```diff
@@ -21,10 +21,30 @@ public enum NuimoGesture
         SwipeRight,
         SwipeUp,
         SwipeDown,
+        TouchLeft,
+        TouchRight,
+        //TouchTop seems not to work
```

Comment:
Yes, we don't make it yet available since touch-top (de-)activates the fly sensor. The event is however reserved in case of future use.